src/pages/api/web3mail/fetch-email-amount.ts

```diff
@@ -0,0 +1,25 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import mongoose from 'mongoose';
+import { fetchSentEmailAMount } from '../../../modules/Web3mail/utils/database';
+
```

Comment:
weird to have 'fetch' in the path of an api. 
This api could be more generic i think: web3mail/stats
for now we just return a simple objectif with total and later we will add lot of different info